build/media-suite_compile.sh

```diff
@@ -1285,15 +1285,13 @@ if enabled libxvid && [[ $standalone = y ]] && ! { files_exist "${_check[@]}" &&
 fi
 
 _check=(libvmaf.{a,h,pc})
-if [[ $ffmpeg != "no" && $bits = 64bit ]] && enabled libvmaf &&
-    do_vcs "https://github.com/Netflix/vmaf.git"; then
+if [[ $ffmpeg != "no" && $bits = 64bit ]] || do_removeOption --enable-libvmaf &&
```

Comment:
At least use `{ }`. This surely doesn't work better than the previous code.

Reply:
Too late now. Anyway, it works better, I had tested it several times.